### PR TITLE
Bug - hasColumn

### PR DESCRIPTION
**Pull request by V3nn3tj3**

### CODE REVIEW

File: src/Backend/Core/Engine/Meta.php [hunk 1]
```diff
@@ -477,6 +477,11 @@ public function save($update = false)
             $custom = null;
         }
 
+        //serialize data for save
+        if (!empty($this->data['data'])) {
+            $this->data['data'] = serialize($this->data['data']);
```
Comment (carakas) [inline src/Backend/Core/Engine/Meta.php]: isn't this a part of an other PR

Comment (V3nn3tj3) [inline src/Backend/Core/Engine/Meta.php]: Yes, it is.
Not a genius when it comes to git, so don't know what happend or how to fix it :(

Comment (carakas) [inline src/Backend/Core/Engine/Meta.php]: The problem is that you start from your own master branch and add your fixes to it, it is better to create a seperate branch for each pull request you want to make and keep your master in sync with the master of fork

Comment (carakas) [inline src/Backend/Core/Engine/Meta.php]: that way your code will come into master when it is merged and won't come in other PR you make

Comment (V3nn3tj3) [inline src/Backend/Core/Engine/Meta.php]: Thanks for the info, are there any other 'tricks'? Can i cancel this PR and create a new one from a seperate branch?

Comment (WouterSioen) [inline src/Backend/Core/Engine/Meta.php]: you could checkout to master, create a new branch, and then cherry-pick the needed commits in there.

**Pull request closed without merging**